docs/administration-guide/named-operations.md

```diff
@@ -335,6 +335,114 @@ EntitySeed[vertex=3]
 ```
 For more examples of Named Operations, please refer to the [Named Operations page in the Reference Guide](../reference/operations-guide/named.md).
 
+### Nested Named Operations
+
+Nested Named Operations allow users to run stored Named Operations within one another. 
+To enable this feature users must set the [store property](../administration-guide/gaffer-stores/store-guide.md#all-general-store-properties)
+`gaffer.named.operation.nested` to true.
+
+There is a [default depth limit](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/graph/hook/NamedOperationResolver.html#DEPTH_LIMIT_DEFAULT) 
+set on nested Named Operations of 3.
```

Comment:
```suggestion
The [default depth limit](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/graph/hook/NamedOperationResolver.html#DEPTH_LIMIT_DEFAULT) 
for nested Named Operations is 3.
```
I'm not sure that we need a link to the code, as looking at this doesn't really give an admin any useful information.